sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -68,11 +51,22 @@
     private final String readAllConflictsSpanName;
     private final String queryConflictsSpanName;
     private CosmosAsyncScripts scripts;
+    private final boolean bulkExecutionEnabled;
+    private BatchAsyncContainerExecutor executor;
```

Comment:
nit final

Reply:
This is because executor wont be initialized if bulk is not enabled and final needs an init. Will it be ok to do?:
if (this.bulkExecutionEnabled) {
            this.executor = new BatchAsyncContainerExecutor(
                this,
                MAX_OPERATIONS_IN_DIRECT_MODE_BATCH_REQUEST,
                MAX_DIRECT_MODE_BATCH_REQUEST_BODY_SIZE_IN_BYTES);
        } else {
            this.executor = null;
        }